packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## TBD Release
+
+- Bugfix: Export new Proxy class from Zowe imperative package.
```

Comment:
For CLI repo we have a different changelog header. Could you please update it and add a link to issue/PR?
```suggestion
## Recent Changes

- BugFix: Export new Proxy class from Zowe imperative package. [#2205](https://github.com/zowe/zowe-cli/pull/2205)
```

Reply:
thanks I found it in Contributing Guidelines too 😄